username_api.py

```diff
@@ -118,9 +144,33 @@ def check_username(website, username):
             'url': url,
             'avatar': get_avatar(website, username),
             'possible': possible,
+            'usable': usable,
         }
 
 
+def check_usable(website):
```

Comment:
just one pick: can you describe the need for this function. just expand a little in the docstring